src/utils/transformBundle.ts

```diff
@@ -15,7 +15,7 @@ export default function transformBundle (
 		return promise.then(code => {
 			return Promise.resolve()
 				.then(() => {
-					return plugin.transformBundle(code, { format: options.format });
+					return plugin.transformBundle(code, options);
```

Comment:
I've intentionally not whitelisted `sourcemap` like `format`, because now `plugin.options` can only get inputOptions if Node API is used, so it makes sense to expose full output options in transformBundle to also cover for future usecases.

Not sure if this is against Rollup's plugin architecture/philosophy. If it is I can whitelist sourcemap and pass it down.